converter/internal/staticconvert/staticconvert_test.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 func TestConvert(t *testing.T) {
-	test_common.TestDirectory(t, "testdata", ".yaml", true, []string{}, staticconvert.Convert)
-	test_common.TestDirectory(t, "testdata-v2", ".yaml", true, []string{"-enable-features", "integrations-next"}, staticconvert.Convert)
+	test_common.TestDirectory(t, "testdata", ".yaml", true, []string{"-config.expand-env"}, staticconvert.Convert)
```

Comment:
This is no longer hard coded to be added so we need to pass it in for the tests.